package.json

```diff
@@ -9,14 +9,16 @@
   ],
   "type": "commonjs",
   "main": "index.js",
+  "types": "types/index.d.ts",
   "exports": {
     ".": "./index.js",
     "./icons": {
       "module": "./icons.mjs",
       "import": "./icons.mjs",
       "require": "./icons.js",
       "default": "./icons.js"
-    }
+    },
+    "./icons/*": "./icons/*.js"
```

Comment:
I missed this in the last PR, this is necessary for Node >= 12 to import those files.